.gitignore

```diff
@@ -30,3 +30,9 @@ test/test/obj
 *~
 .#.*
 core
+# CMake output
+CMakeFiles
+CMakeCache.txt
+cmake_install.cmake
```

Comment:
`CMakeFiles`, `CMakeCache.txt` and `cmake_install.cmake` and friends shouldn't be present here since they should only be in build directories. I tend to always build in a separate build directory and it doesn't even have to be under the root mps directory.